software/control/gui_hcs.py

```diff
@@ -477,8 +473,6 @@ def setup_hardware(self):
 
             self.microscope.home_xyz()
```

Comment:
For some stages may not want to do homing at startup (e.g. Prior stage on Ti2-U) - how is that taken care of here?